stable/elasticsearch-exporter/templates/service.yaml

```diff
@@ -19,6 +19,10 @@ spec:
   ports:
     - name: {{ .Values.service.metricsPort.name }}
       port: {{ .Values.service.httpPort }}
+{{- if eq .Values.service.type "NodePort"}}
```

Comment:
imho this and the `end` should be in the same indent level as `port` above

Reply:
done